components/Tabs/Tabs.js

```diff
@@ -1,61 +1,59 @@
 class TabLink {
-  constructor(tabElement){
+  constructor(tabElement) {
     // assign this.tabElement to the tabElement DOM reference
-    // this.tabElement;
-    
+    this.tabElement = tabElement;
+
     // Get the `data-tab` value from this.tabElement and store it here
-    // this.tabData = ; 
-    
+    this.tabData = this.tabElement.dataset.tab; //wondering if I should add the "this." 
```

Comment:
you are still in the constructor so you can do either. You just assigned this.tabEelement so you have access to it. But you also have access to just tabElement that was passed in.  So its really up to you.